math/polycalc/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+github.setup        Zeta611 polycalc 8c6922fab48f8e23ac4f02d00cd27cb271a02965
+version             2023.01.03
+revision            0
+categories          math
+license             MIT
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         PolyCalc: Polynomial Calculator
+long_description    PolyCalc is a polynomial calculator. It calculates and expands polynomials, equations and relations. \
+                    It supports simple variable assignments for the ease of handling expressions.
+checksums           rmd160  54221d07103237afc407a791e5d0b725fc2da7a9 \
+                    sha256  de11c920f403d8a87f9415d11df7eb8cdc212ea799f13b3b4f0f24bf31f6fc17 \
+                    size    12422
+
+depends_build-append \
+                    port:bison \
+                    port:flex
+
+compiler.c_standard 2017
+
+post-patch {
+    reinplace "s,gcc,${configure.cc}," ${worksrcpath}/Makefile
+}
+
+build.cmd           make
```

Comment:
```suggestion
```

I think `build.cmd` should already be set as make.

Reply:
Did not work for me initially if removed, but let me try again (there was a fix with upstream for our clang builds, I am not sure if I tried to remove this line after that).

Reply:
@harens No, it fails without this. I dunno why :) But I normally do not add something that should be automatic without a reason (_well, I try to, at least_).
Here is what happens with both build settings removed:
```
--->  Configuring polycalc
--->  Building polycalc
Executing:  cd "/opt/local/var/macports/build/_opt_PPCRosettaPorts_math_polycalc/polycalc/work/polycalc-8c6922fab48f8e23ac4f02d00cd27cb271a02965" && /usr/bin/make -j8 -w all CC="/opt/local/var/macports/build/_opt_PPCRosettaPorts_math_polycalc/polycalc/work/compwrap/cc/opt/local/bin/gcc-mp-12" CXX="/opt/local/var/macports/build/_opt_PPCRosettaPorts_math_polycalc/polycalc/work/compwrap/cxx/opt/local/bin/g++-mp-12" OBJC="/opt/local/var/macports/build/_opt_PPCRosettaPorts_math_polycalc/polycalc/work/compwrap/objc/opt/local/bin/gcc-mp-12" OBJCXX="/opt/local/var/macports/build/_opt_PPCRosettaPorts_math_polycalc/polycalc/work/compwrap/objcxx/opt/local/bin/g++-mp-12" FC="/opt/local/var/macports/build/_opt_PPCRosettaPorts_math_polycalc/polycalc/work/compwrap/fc/opt/local/bin/gfortran-mp-12" F77="/opt/local/var/macports/build/_opt_PPCRosettaPorts_math_polycalc/polycalc/work/compwrap/f77/opt/local/bin/gfortran-mp-12" F90="/opt/local/var/macports/build/_opt_PPCRosettaPorts_math_polycalc/polycalc/work/compwrap/f90/opt/local/bin/gfortran-mp-12" INSTALL="/usr/bin/install -c" 
make: Entering directory `/opt/local/var/macports/build/_opt_PPCRosettaPorts_math_polycalc/polycalc/work/Zeta611-polycalc-8c6922f'
make: *** No rule to make target `all'.  Stop.
```
And this with build.cmd removed and build.target retained:
```
./build/src/poly.tab.c:372:54: error: token "\" is not valid in preprocessor expressions
  372 | #  if (defined __cplusplus && ! defined EXIT_SUCCESS \
      |                                                      ^
./build/src/poly.tab.c:372: error: unterminated #if
  372 | #  if (defined __cplusplus && ! defined EXIT_SUCCESS \
      | 
./build/src/poly.tab.c:356: error: unterminated #else
  356 | # ifdef YYSTACK_ALLOC
      | 
./build/src/poly.tab.c:328: error: unterminated #if
  328 | #if !defined yyoverflow
      | 
make: *** [build/./src/poly.y.o] Error 1
make: *** Waiting for unfinished jobs....
```
With both in place, it builds fine.

Reply:
Fair enough, thanks for checking :)

Reply:
@harens I honestly stopped understanding this build. I tried today to rebuild it few times, and it did work without patch and most of manual settings. No idea what changed. For now, I removed extra settings, let us see if it builds on bots.
(Now CI are broken due to unrelated issue.)

Reply:
It's all been very unusual. I have no idea why it sometimes builds and then fails when nothing's changed. Thanks for persisting though.